test/worker.spec.js

```diff
@@ -15,7 +15,7 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(await worker.evaluate(() => self.workerFunction())).toBe('worker function result');
 
       await page.goto(server.EMPTY_PAGE);
-      expect(page.workers()).toEqual([]);
+      expect(page.workers().length).toBe(1);
```

Comment:
This doesn't make sense to me. Isn't it the opposite of what the test is testing for? EMPTY_PAGE shouldn't have any workers.

Reply:
Ah, I meant 0.